draft-ietf-quic-transport.md

```diff
@@ -4229,39 +4224,37 @@ stream data that it has sent for the affected stream, unless this is a result of
 a change in the initial limits (see {{zerortt-parameters}}).
 
 
-## MAX_STREAM_ID Frame {#frame-max-stream-id}
+## MAX_STREAMS Frame {#frame-max-streams}
 
-The MAX_STREAM_ID frame (type=0x06) informs the peer of the maximum stream ID
-that they are permitted to open.
+The MAX_STREAMS frame (type=0x1c and 0x1d) informs the peer of the number of
+streams it is permitted to open.  A MAX_STREAMS frame with a type of 0x1c
+applies to bidirectional streams; a MAX_STREAMS frame with a type of 0x1d
+applies to unidirectional streams.
```

Comment:
Would it be clearer to frame this as a flag?

Reply:
I just tried this.  Describing using a flag means more words, and I think that it's a small regression in terms of clarity.

Reply:
Yeah, I'm not thrilled with this either. How about we separate the frame types in the description? These are different frames, and I think it might be clearer to simply call them MAX_STREAMS_UNI and MAX_STREAMS_BIDI.

Reply:
After having a look at #1900, I think its going to be hard to avoid this clunkiness. This PR is a net positive and combining these frames does improve readability, so I'll drop my suggestion.

Reply:
Thanks Jana.  There's no truly good way of dealing with this, but I think that we've at least hit a local minimum.